how-to/divio-app.rst

```diff
@@ -0,0 +1,126 @@
+.. _divio-app:
+
+How to install and use the Divio app
+====================================
+
+The Divio app helps you run your Divio projects locally in our Docker environment, upload and download media, database
+and configuration changes as well as set up a shell environment ready for interaction with the project running in the
+Docker container.
+
+
+Set up the Divio app
+--------------------
+
+
+Installation
+~~~~~~~~~~~~
+
+* **Download** the `Divio app here <https://divio.com/developers/#app>`_, which is available for Macintosh, Linux and
+  Windows.
+
+* Open the downloaded package and follow the on-screen instructions to install the application.
+
+.. _note:: 
+
+  Some anti-virus software for Windows may warn that the Divio app application is unsigned and therefore represents a
+  security risk. You can safely ignore this warning.
+
+
+Loging in
+~~~~~~~~~
+
+Once the application is installed and running you can login using your email address and password, or if you singed up
+with Google or GitHub, simply copy `the access token <https://control.divio.com/account/desktop-app/access-token/>`_
+from the Divio control panel and paste it into the provided field.
+
+Select a workspace folder on your computer where you’d keep all the Divio cloud project files.
+
+
+Use the Divio app
+-----------------
+
+Select a project to work with from the list. By default, the list shows all projects, but you can narrow down the
+selection by choosing a particular organisation from the dropdown menu.
+
+
+Set up your local project
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Select a project from the list.
+* Hit *set up project* to get started.
+
+The Divio app will run through a number of processes to set up the local project. It all happens automatically and takes
+just a few minutes.
+
+
+Launch the site locally
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the set up process has completed, you can start the local site by launching the local server. 
+
+* Hit **Start** to launch the project.
+* If the project has been deployed, you can now open the local site in your browser by clicking on the *eye icon*.
```

Comment:
whether or not it has been deployed has no effect really on this, i'd say something like "If the project has been set up correctly, you can now...". Most of the time it wouldn't start due to some sort of database failure, and it's fine to say that in a note, but better to be explicit about that.

Reply:
For now, I changed the "deployed" to "set up correctly"